hc/accounts/tests/test_check_token.py

```diff
@@ -15,14 +15,29 @@ def test_it_shows_form(self):
 
     def test_it_redirects(self):
         r = self.client.post("/accounts/check_token/alice/secret-token/")
+        self.assertEqual(r.status_code, 302)
+        # self.assertIn(self.profile.user, self.client.session)
         self.assertRedirects(r, "/checks/")
 
-        # After login, token should be blank
+     # After login, token should be blank
         self.profile.refresh_from_db()
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_redirects_already_logged_in(self):
```

Comment:
***Code Quality: Nitpick***
this should be written like so
`def test_redirects_if_already_logged_in`